core/src/main/scala/stryker4s/Stryker4s.scala

```diff
@@ -7,14 +7,16 @@ import stryker4s.mutants.Mutator
 import stryker4s.run.MutantRunner
 import stryker4s.run.threshold.{ScoreStatus, ThresholdChecker}
 
-class Stryker4s(fileSource: MutatesFileResolver, mutator: Mutator, runner: MutantRunner)(implicit config: Config) {
+class Stryker4s(fileSource: MutatesFileResolver, mutatorFactory: () => Mutator, runner: MutantRunner)(implicit
```

Comment:
Is this mutatorFactory still needed? I couldn't find anywhere where it's called a second time

Reply:
It's no longer needed for the rollback logic (which is why I accidentally stopped calling twice). I removed it.

It doesn't matter functionally, but now in the case of a rollback the ids no longer start at 0, but I don't think that's a huge deal. If you disagree let me know and I'll add it back.

Reply:
Done in c77d70d